apps/web-staking/src/app/components/redeem/History.tsx

```diff
@@ -269,7 +268,7 @@ export default function History({ redemptions, reloadRedemptions }: {
 									claimable={true}
 									claimDisabled={isLoading}
 									receivedAmount={r.receiveAmount}
-									redeemedAmount={r.redeemAmount}
+									redeemedAmount={r.amount}
```

Comment:
Was there any specific reason this got renamed to `amount` instead of ``redeemAmount` ?

Reply:
It was so to align with the return types from the smart contract. The struct in the contract calls it amount and it was causing type errors with the previous name.